care/facility/migrations/0415_patientconsultation_previous_consultation.py

```diff
@@ -0,0 +1,51 @@
+# Generated by Django 4.2.10 on 2024-02-18 06:37
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("facility", "0414_remove_bed_old_name"),
+    ]
+
+    def fill_previous_consultation(apps, schema_editor):
+        PatientConsultation = apps.get_model("facility", "PatientConsultation")
+        bulk_update_list = []
+        qs = PatientConsultation.objects.all()
```

Comment:
```suggestion
        qs = PatientConsultation.objects.all().order_by("-created_date")
```

Reply:
@sainak I suppose this is for performance only? don't think it will change any behavior.